<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>745</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>745</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>745</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Face Recognition Attendance App</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>24</y>
     <width>673</width>
     <height>625</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>4</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>660</y>
     <width>673</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ClockInButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>CocogooseProTrial Semilight</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: rgb(105, 248, 255);
    border-radius: 5px;
	color: rgb(9, 28, 84);
	font: 10pt &quot;CocogooseProTrial Semilight&quot;;

}

QPushButton:hover {
	background-color: rgb(137, 227, 254);
}

QPushButton:pressed {
background-color: rgb(105, 248, 255);
}


</string>
      </property>
      <property name="text">
       <string>CLOCK IN</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClockOutButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>CocogooseProTrial Semilight</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: rgb(105, 248, 255);
    border-radius: 5px;
	color: rgb(9, 28, 84);
	font: 10pt &quot;CocogooseProTrial Semilight&quot;;

}

QPushButton:hover {
	background-color: rgb(137, 227, 254);
}

QPushButton:pressed {
background-color: rgb(105, 248, 255);
}


</string>
      </property>
      <property name="text">
       <string>CLOCK OUT</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>718</x>
     <y>223</y>
     <width>383</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 128);
color: rgb(255, 255, 255);
font: 57 10pt &quot;Montserrat Medium&quot;;
border-radius:5px;</string>
   </property>
   <property name="title">
    <string>Details</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>36</y>
      <width>205</width>
      <height>229</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Montserrat SemiBold</family>
         <pointsize>14</pointsize>
         <weight>7</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
color: rgb(14, 23, 67);
padding: 5px;</string>
       </property>
       <property name="text">
        <string>Name : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Montserrat SemiBold</family>
         <pointsize>14</pointsize>
         <weight>7</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
color: rgb(14, 23, 67);
padding: 5px;</string>
       </property>
       <property name="text">
        <string>Status :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Montserrat SemiBold</family>
         <pointsize>14</pointsize>
         <weight>7</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
color: rgb(14, 23, 67);
padding: 5px;</string>
       </property>
       <property name="text">
        <string>Clocked Time : </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>36</y>
      <width>145</width>
      <height>229</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="NameLabel">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>10</pointsize>
         <weight>7</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="StatusLabel">
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>10</pointsize>
         <weight>7</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="HoursLabel">
         <property name="font">
          <font>
           <family>Montserrat Medium</family>
           <pointsize>10</pointsize>
           <weight>7</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="MinLabel">
         <property name="font">
          <font>
           <family>Montserrat Medium</family>
           <pointsize>10</pointsize>
           <weight>7</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>718</x>
     <y>530</y>
     <width>383</width>
     <height>190</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 128);
color: rgb(255, 255, 255);
font: 57 10pt &quot;Montserrat Medium&quot;;
border-radius:5px;</string>
   </property>
   <property name="title">
    <string>User Functions</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>36</y>
      <width>349</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonRegist">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: qlineargradient(spread:pad, x1:0.134328, y1:0.182, x2:1, y2:0, stop:0 rgba(12, 33, 158, 255), stop:1 rgba(26, 72, 255, 255));
	color: rgb(255, 255, 255);
    border-radius: 5px;
	font: 57 10pt &quot;Montserrat Medium&quot;;

}

QPushButton:hover {
	
	background-color: rgb(29, 60, 199);

}

QPushButton:pressed {
    background-color: #0581F9; /* Màu khi nhấn */
}

</string>
       </property>
       <property name="text">
        <string>Register Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonManage">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: qlineargradient(spread:pad, x1:0.134328, y1:0.182, x2:1, y2:0, stop:0 rgba(12, 33, 158, 255), stop:1 rgba(26, 72, 255, 255));
	color: rgb(255, 255, 255);
    border-radius: 5px;
	font: 57 10pt &quot;Montserrat Medium&quot;;

}

QPushButton:hover {
	
	background-color: rgb(29, 60, 199);

}

QPushButton:pressed {
    background-color: #0581F9; /* Màu khi nhấn */
}

</string>
       </property>
       <property name="text">
        <string>Management</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>745</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../background1.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>718</x>
     <y>25</y>
     <width>385</width>
     <height>193</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>14</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
color: rgb(12, 33, 158);
padding: 5px;</string>
      </property>
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>14</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
color: rgb(12, 33, 158);
padding: 5px;</string>
      </property>
      <property name="text">
       <string>Time :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="Date_Label">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>14</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 14pt &quot;Montserrat ExtraBold&quot;;
color: rgb(12, 33, 158);
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="Time_Label">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>14</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 14pt &quot;Montserrat ExtraBold&quot;;
color: rgb(12, 33, 158);
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_6</zorder>
  <zorder>imgLabel</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>gridLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
